en/developer/frameworks/articles/search/03-queries-and-filters/02-building-search-queries-and-filters.markdown

```diff
@@ -90,22 +104,23 @@ Once the query itself is in good shape, feed it to the search request.
     `com.liferay.portal.kernel.search.SearchContext`:
 
     ```java
-    searchRequestBuilder.withSearchContext(
-                searchContext -> searchContext.setCompanyId(companyId);
-                searchContext -> searchContext.setKeywords(keywords);
-            );
+		searchRequestBuilder.withSearchContext(
```

Comment:
was this indent intentional?

Reply:
Please check how it looks like in the Java class where it was copied from. That's passing SF.

Reply:
Ahh, I see. These are inline (markdown) code blocks, so the indentation is determined by the context of the markdown, not a Java class. The example code below can be properly formatted for copy and pasting.

I believe that will give us consistency with other articles.

Reply:
You can remove the indents from the individual snippets if it looks weird, I copied them from their actual context from the complete example.